src/main/kotlin/graphql/kickstart/tools/MissingFieldResolverHandler.kt

```diff
@@ -0,0 +1,7 @@
+package graphql.kickstart.tools
+
+import graphql.schema.DataFetchingEnvironment
+
+interface MissingFieldResolverHandler {
```

Comment:
Any chance we could let the user pass in the whole `graphql.schema.DataFetcher`? I feel like this interface is the same thing and just adds one layer of indirection. Let me know your thoughts.

Reply:
Yep that makes sense, pushed another commit to allow the consumer to directly provide the `DataFetcher<Any?>`.